aroma/utils.py

```diff
@@ -301,7 +301,7 @@ def classification(features_df, out_dir):
                                 components identified as motion components
     """
     # Put the feature scores in a text file
-    features_df.to_csv(op.join(out_dir, "feature_scores.tsv"), sep="\t", index=False)
+    features_df.to_csv(op.join(out_dir, "feature_scores.txt"), sep="\t", index=False)
```

Comment:
Why make this change? We want tsv files for BIDS Derivatives compatibility.

Reply:
Sorry, I see that I said "change the extension" when this came up earlier. I just meant that you should update the _test_, not the file.

Reply:
ohhh you wanted me to do the change on the test script?

Reply:
Yes, sorry. My original comment was poorly phrased.

Reply:
Sure i will make the change